src/main/java/seedu/address/model/task/TaskList.java

```diff
@@ -113,20 +115,41 @@ public void setTask(Task taskToEdit, Task editedTask) {
         return tasksUnmodifiable;
     }
 
-    public List<Task> getSortedList(Comparator<Task> comparator) {
+    public static List<Task> getSortedList(List<Task> tasks, Comparator<Task> comparator) {
```

Comment:
This method is probably not needed, it's just `tasks.sort(comparator)`, don't need to use streams also.

Reply:
solved